README.md

```diff
@@ -1,4 +1,4 @@
-![banner](docs/edit/images/banner.png)
+****![banner](docs/edit/images/banner.png)
```

Comment:
Why did you add "****" ?

Reply:
Resolved! I'm sure I just hit command+b by accident.

Reply:
You're welcome.